internal/web/web.go

```diff
@@ -163,6 +168,14 @@ func (web *Web) GetSSOCredentials(conf credentialexchange.CredentialConfig) (str
 	}
 }
 
+func (web *Web) MustClose() {
+	err := web.browser.Close()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%s", err)
+	}
+	// checkRodProcess()
```

Comment:
@dnitsch Hmmm?

Reply:
this would kill all chromium instances... not sure we want that at this point